com.blackmagicdesign.Resolve.yml

```diff
@@ -0,0 +1,186 @@
+id: com.blackmagicdesign.Resolve
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: resolve.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-cache
+  - --filesystem=xdg-data
+  - --filesystem=xdg-videos
+  - --filesystem=~/.local/share/DaVinciResolve
```

Comment:
this should be persist if it is actually the path it uses instead of follow the XDG standard proper